src/app/core/enums/add-answer-click-count.enum.ts

```diff
@@ -0,0 +1,3 @@
+export enum AddAnswerClickCountEnum {
```

Comment:
Don't need enum for one param. Create smth like "constants" and set all minor constants there